app/code/Magento/Backend/Block/Widget/Grid/Column/Renderer/AbstractRenderer.php

```diff
@@ -55,9 +55,6 @@ public function render(DataObject $row)
     {
         if ($this->getColumn()->getEditable()) {
             $result = '<div class="admin__grid-control">';
-            $result .= $this->getColumn()->getEditOnly() ? ''
```

Comment:
Hi @speedy008. Thanks for collaboration. I am not sure that removing code is a valid fix for this issue. I see that here we have a checking `getEditOnly()`. Maybe the reason of this problem that the value returned from `getEditOnly` is wrong?

Also, it's abstract renderer class and I am not sure that fix should be in this place. This class is using in a lot of places, but problem we have only on the `Assign products` grid

Reply:
Hi @VladimirZaets 
Thanks for your genuine response.
I will update code with proper solution and update it as early as possible.